EasyPost.Tests/ModelsTests/TrackerTest.cs

```diff
@@ -0,0 +1,43 @@
+using System.Threading.Tasks;
+using EasyPost.Models.API;
+using EasyPost.Tests._Utilities;
+using EasyPost.Tests._Utilities.Annotations;
+using Xunit;
+
+namespace EasyPost.Tests.ModelsTests
+{
+    public class TrackerTests : UnitTest
+    {
+        public TrackerTests() : base("tracker")
+        {
+        }
+
+        [Fact]
+        [Testing.Properties]
+        public async Task TestTrackerCarrierDetails()
+        {
+            // Details can be (and most likely will be) missing or incomplete.
```

Comment:
Therefore... what? Can we elaborate on this? Are we basically saying "because stuff is missing, we can't test it easily" here? If so, let's say that.

Reply:
Resolved